documentation/modules/auxiliary/scanner/http/apache_userdir_enum.md

```diff
@@ -0,0 +1,19 @@
+## Description
+
+This module determines if usernames are valid on a server running Apache with the UserDir directive enabled. 
+It takes advantage of Apache returning different error codes for usernames that do not exist and for usernames with no `public_html` directory.
+
```

Comment:
Assuming this isn't on by default, can you add how to enable it?
 Put in an L3 heading (triple #) with which OS youre showing how to enable it on. Something like
```
### Enabling on Ubuntu
1. `sudo a2enmod userdir`
2. Edit `/etc/apache2/apache.conf` and add `UserDir enabled user1 user2 user3`
3. `sudo apache2ctl restart`
```